test/Olcs/src/FormService/Form/Lva/People/SoleTrader/LicenceSoleTraderTest.php

```diff
@@ -218,7 +148,7 @@ public function testGetFormCantModify()
             ->once()
             ->with($form, 'bar');
 
-        $this->sm->setService('Lva\People', $this->peopleLvaService);
+        $this->fsm->setService('Lva\People', $this->peopleLvaService);
```

Comment:
Is this mock used? I raise this because we're moving where we are mocking an object without any source code changes which implies that it's not called.